code/modules/antagonists/monsterhunter/monstertrack.dm

```diff
@@ -6,7 +6,7 @@
 	icon_icon = 'icons/mob/actions/actions_bloodsucker.dmi'
 	background_icon_state = "vamp_power_off"
 	button_icon_state = "power_hunter"
-	power_flags = NONE
+	power_flags = BP_AM_TOGGLE
```

Comment:
Why is this ability still a toggle?

Reply:
toggle actually works better than whatever it was doing before with NONE

Reply:
Ok but why is it still a toggle

Reply:
toggle auto updates the power when it is deactivated, with no flags it actually just keeps the background unupdated